Library/Homebrew/cmd/link.rb

```diff
@@ -71,11 +73,10 @@ def link
     end
   end
 
-  private
-
   def keg_only?(rack)
     Formulary.from_rack(rack).keg_only?
   rescue FormulaUnavailableError, TapFormulaAmbiguityError, TapFormulaWithOldnameAmbiguityError
     false
   end
+  private_class_method :keg_only?
```

Comment:
I think for the `cmd`s we can remove the `private_class_method`s as we don't really care if they are private or not.